packages/jats-xml/src/resolvers.ts

```diff
@@ -0,0 +1,58 @@
+import checkdoi from 'doi-utils';
+import fetch from 'node-fetch';
+import type { ISession } from './types';
+
+export interface Resolver {
+  test: (url: string) => boolean;
+  jatsUrl: (url: string) => string;
+}
```

Comment:
Right now only have three resolvers, I think we should follow the advice of Alf and look to cross ref and see if it is there.

https://twitter.com/invisiblecomma/status/1596095274828824576

> If the publisher has deposited a URL for the article XML with Crossref, yes:
> https://www.biglist.com/lists/lists.mulberrytech.com/jats-list/archives/202204/msg00009.html
> https://www.biglist.com/lists/lists.mulberrytech.com/jats-list/archives/202204/msg00010.html

That can probably happen in a next step.